docker-compose-eventuate-local-mysql.yml

```diff
@@ -102,7 +101,7 @@ services:
     build:
       context: ./orders-service/
       args:
-        baseImageVersion: ${EVENTUATE_JAVA_BASE_IMAGE_VERSION?}
+        baseImageVersion: ${EVENTUATE_JAVA_BASE_IMAGE_VERSION}
```

Comment:
Why is '?' removed?

Reply:
Because it is not possible to see docker logs without setting this variable. But variable itself is not useful, because in case if it is not set docker build will just fail.